Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -2566,6 +2701,11 @@ internal bool LoadingOrUnloadingRow
             private set;
         }
 
+        internal void LoadMoreDataFromIncrementalItemsSource()
```

Comment:
This should be moved among the internal methods, just above "internal void OnRowDetailsChanged()"